Source/Core/Parser.cs

```diff
@@ -428,6 +428,8 @@ public override int ComputeHashCode()
 		 // generate either an axiom or a function body
 		 if (QKeyValue.FindBoolAttribute(kv, "inline")) {
 		   func.Body = definition;
+		 } else if (QKeyValue.FindBoolAttribute(kv, "define")) {
+		   ds.Add(func.CreateFunctionDefinition(definition, kv));
```

Comment:
@bkragl : I recall that you explained that Parser.cs is a generated file and should not be modified.  Could you please add the appropriate steps here when modifying BoogiePL.atg?

Reply:
`Parser.cs` is generated, but we still keep it in the repository (I guess to avoid parser generation as part of the build process). I see that this change comes from `BoogiePL.atg` so everything is good.